verification-http/src/request.rs

```diff
@@ -308,8 +308,9 @@ mod test {
         let query_params = HashMap::new();
         let path_params = HashMap::new();
 
+        let body_length = json.len() as u64 + 1;
         let mut request = Request::new(&path_params, &query_params, &headers, &mut json);
-        request.body_size_limit = json.len() as u64 + 1;
+        request.body_size_limit = body_length;
```

Comment:
this is to make the project compile with rust nightly 1.41.0. It complains about immutable borrow after mutable borrow otherwise